.travis-deps.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if [ $TRAVIS_OS_NAME = "osx" ]; then
+    brew update
+    brew install flex bison libzip pkg-config
```

Comment:
BTW, `flex` and `bison` should already be installed, so there shouldn't be any need to `brew install` these.

Also, is there any reason for `libzip`? I've not got it installed on my Mac.